utils/components-testkit/src/main/scala/pl/touk/nussknacker/engine/util/test/ModelWithTestComponents.scala

```diff
@@ -4,15 +4,19 @@ import com.typesafe.config.Config
 import com.typesafe.config.ConfigValueFactory.fromAnyRef
 import pl.touk.nussknacker.engine.ModelData
 import pl.touk.nussknacker.engine.api.component.ComponentDefinition
-import pl.touk.nussknacker.engine.api.process.EmptyProcessConfigCreator
+import pl.touk.nussknacker.engine.api.process.{EmptyProcessConfigCreator, WithCategories}
 import pl.touk.nussknacker.engine.testing.LocalModelData
 
 object ModelWithTestComponents {
 
   // we add components by hand, which should be tested in scenario
   // components are registered in special ComponentProvider, which is configured with appropriate testRunId
-  def withTestComponents[T](config: Config, components: List[ComponentDefinition])(action: ModelData => T): T = {
-    val testComponentHolder = TestComponentsHolder.registerTestComponents(components)
+  def withTestComponents[T](
```

Comment:
I see "components" in this method name and `globalVariables` are not components, the same with `TestComponentsHolder`. Have you and idea how to handle that?

Reply:
ModelWithTestElements -> withTestElements?

Reply:
`ModelWithTestExtensions` maybe?

Reply:
Done